src/components/intern-app.html

```diff
@@ -1,137 +1,179 @@
-              return [
-                { id: "001", firstName: 'Nick', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "002", firstName: 'Jack', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "003", firstName: 'Betty', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "004", firstName: 'Jordan', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "005", firstName: 'Brian', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "006", firstName: 'Gabby', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "007", firstName: 'John', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "008", firstName: 'Tony', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "009", firstName: 'Ryan', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "010", firstName: 'Mack', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" }
-              ];
-            }
+     static get properties() {
+       return {
+         userProfiles: {
+           type: Array,
+           value: () => localStorageApi.getUsers()
+         },
+         addUserSelected: {
+           type: Boolean,
+           value: false
+         },
+         isExpanded: {
+            type: Boolean,
+            value: true
           },
-          addUserSelected: {
+         cardsPopulated: {
             type: Boolean,
             value: false
-          }
-        }
-      }
+         }
+       }
+     }
 
-      constructor() {
-        super();
-        this.boundNewUser = this.newUser.bind(this);
-      }
+     constructor() {
+       super();
+       this.boundNewUser = this.newUser.bind(this);
+     }
 
-      connectedCallback() {
-        super.connectedCallback();
-        window.addEventListener("newUser", this.boundNewUser);   
-      }
+     connectedCallback() {
+       super.connectedCallback();
+       window.addEventListener("newUser", this.boundNewUser);
+     }
 
-      disconnectedCallback() {
-        super.disconnectedCallback();
-        window.removeEventListener("newUser", this.boundNewUser); 
-      }
+     disconnectedCallback() {
+       super.disconnectedCallback();
+       window.removeEventListener("newUser", this.boundNewUser);
+     }
 
-      newUser(event) {
-        var newUserProfile = Object.assign({}, event.detail);
-        newUserProfile.id = this.genId();
-        this.pushToArray(this.userProfiles, newUserProfile);
-        this.addUserSelected = false;
-      }
+     deleteAll() {
+      if (window.confirm("Are you sure you want to delete all users?")) {
+        localStorageApi.deleteUsers();
+        cardsPopulated = false;
```

Comment:
need to make sure to use `this.` for all properties defined in `static get properties`
```suggestion
        this.cardsPopulated = false;
```